src/PoolManager.sol

```diff
@@ -48,6 +52,7 @@ contract PoolManager is Auth, IPoolManager {
 
     IEscrow public immutable escrow;
 
+    address public router;
```

Comment:
I don't see this being used anywhere apart of the `file` method. Probably can be deleted?